onmt/modules/Criterion.lua

```diff
@@ -0,0 +1,30 @@
+--[[
+  Training Criterion
+--]]
+local Criterion, parent = torch.class('onmt.Criterion', 'nn.ParallelCriterion')
+
+
+function Criterion:__init(vocabSize, features)
+  parent.__init(self, false)
+  self:_buildCriterion(vocabSize, features)
+end
+
+function Criterion:_buildCriterion(vocabSize, features)
```

Comment:
Maybe add this to the name or the top comment. This is non-obvious that "Criterion" would do this.